### ISSUE TITLE
If Radex.datapath is not set explicitly, an OSError occurs

### ISSUE THREAD

**Issue by tomr-stargazer** [keflavich/pyradex#11]
Hey,
I just started using pyradex, and wanted to log the following unhandled error.  I was following the example hosted here:
http://nbviewer.jupyter.org/github/keflavich/pyradex/blob/master/examples/FittingTheGrid.ipynb

and when I ran the following line:
```python
R = pyradex.Radex(species='oh2co-h2', abundance=1e-8, column=1e14, temperature=20)
```

I got this error:
https://gist.github.com/tomr-stargazer/9f6448b4bfc5e93011f463bfe6980003

It looks like the code somehow mishandles the fact that self.datapath is set to None, so I wonder if there is a more elegant way to handle this (at least to throw a more sensible error).

I'd be happy to attempt to solve this myself as I get more comfortable and familiar with pyradex!

**Comment by tomr-stargazer** [comment]
The error doesn't come up when I have `$RADEX_DATAPATH` set to something sensible, as expected.

**Comment by keflavich** [comment]
Right, probably the best solution is to raise a descriptive exception earlier if the path is not set, since I think we need the path set in order to stash & retrieve collisional data files.
